integration/default_apps_test.go

```diff
@@ -102,42 +102,64 @@ func testDefaultApps(t *testing.T, context spec.G, it spec.S) {
 		})
 
 		context("given a source application with .NET 7", func() {
-			it("should build (and rebuild) a working OCI image", func() {
+			it("should build a working OCI image", func() {
 				var err error
 				source, err := occam.Source(filepath.Join("testdata", "source_7"))
 				Expect(err).NotTo(HaveOccurred())
 
-				for i := 0; i < 3; i++ {
```

Comment:
what's the reason for this change?

Reply:
This is testing a cache use specific case. It does not feel like it is something that is necessary to test for every version of .NET

Reply:
very well